connecting-to-testnet/index.md

```diff
@@ -9,14 +9,14 @@ description: Start a node, use the Cosmos SDK CLI, and interact with the Cosmos
 
 The [Cosmos Network](https://cosmos.network) is an expanding ecosystem of connected blockchains. Many blockchains in the ecosystem are built with [Tendermint Core](https://docs.tendermint.com/) and [Cosmos SDK](https://docs.cosmos.network/). These blockchains are connected with  the inter-blockchain Communication protocol [(IBC)](https://ibcprotocol.org/). 
 
-One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019. You can learn how to join the network as a validator, see [Run a Validator on the Cosmos Hub Mainnet](https://hub.cosmos.network/main/validators/validator-setup.html). 
+One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019. You can learn how to join the network as a validator, see [Run a Validator on the Cosmos Hub Mainnet](https://hub.cosmos.network/main/validators/validator-setup.html).
 
 ## What is a testnet?
 
 A testnet is a network launched for developers and validators to test the functionality of the blockchain before the main network is launched. The state of a testnet can be reset at any time.
 
-## Tutorial Testnet 
+## Tutorial Testnet
 
-The source code repo `b-harvest/gravity-dex` offers more functionality because it adds the `tendermint/liquidity` module that is the source code of Gravity DEX to the `cosmos/gaia` source code behind Cosmos Hub. 
+The source code repo `b-harvest/gravity-dex` offers more functionality because it adds the `tendermint/liquidity` module that is the source code of Gravity DEX to the `cosmos/gaia` source code behind Cosmos Hub.
```

Comment:
@toschdev I see how you updated the content to point to gaia... does line 20 still make sense? (how would I know? teach me!)

Reply:
Thanks, overlooked this occurence.